website/content/partials/entity-alias-mapping.mdx

```diff
@@ -0,0 +1,6 @@
+## Entity Alias mapping
+
+Previously, an entity in Vault could be mapped to multiple entity aliases on the same authentication backend. This
+led to a potential security vulnerability, as ACL policies templated with alias information, would match the first
+alias created. As a result, the mapping behavior was updated such that an entity can only have one alias per authentication
```

Comment:
This doesn't quite cover the impact of ACL policy templating matching the first alias created - could you please add a statement about the impact before "As a result"?

Reply:
added a note about that here 21edb57, lemme know how it looks!